ovos_local_backend/backend/precise.py

```diff
@@ -1,47 +1,50 @@
 from flask import request
 
+from ovos_local_backend.backend import API_VERSION
 from ovos_local_backend.backend.decorators import noindex, requires_auth, check_selene_pairing
 from ovos_local_backend.configuration import CONFIGURATION
 from ovos_local_backend.database import save_ww_recording
 from ovos_local_backend.utils.selene import upload_ww
 
-
 def get_precise_routes(app):
     @app.route('/precise/upload', methods=['POST'])
     @noindex
     @check_selene_pairing
     @requires_auth
     def precise_upload():
+        success = uploaded = False
         if CONFIGURATION["record_wakewords"]:
             auth = request.headers.get('Authorization', '').replace("Bearer ", "")
             uuid = auth.split(":")[-1]  # this split is only valid here, not selene
-            save_ww_recording(uuid, request.files)
+            success = save_ww_recording(uuid, request.files)
 
-        uploaded = False
         selene_cfg = CONFIGURATION.get("selene") or {}
         if selene_cfg.get("upload_wakewords"):
             # contribute to mycroft open dataset
             uploaded = upload_ww(request.files)
 
-        return {"success": True,
+        return {"success": success,
                 "sent_to_mycroft": uploaded,
                 "saved": CONFIGURATION["record_wakewords"]}
 
-    @app.route('/device/<uuid>/wake-word-file', methods=['POST'])
+    @app.route(f'/{API_VERSION}/device/<uuid>/wake-word-file', methods=['POST'])
```

Comment:
no API version here, the upload is actually a separate service under Mycroft, not part of selene

Reply:
then we have to change this in the backend client, too as this would route

```python
    def device_upload_wake_word(self, audio, params):
        """ upload precise wake word V2 endpoint - integrated with device api"""
        url = f"{self.backend_url}/{self.backend_version}/device/{self.uuid}/wake-word-file"
 ```